.github/workflows/npm-publish-pages.yml

```diff
@@ -4,26 +4,34 @@
 name: Node.js Publish webpage
 
 on:
-  release:
-    types: [created]
+  push:
+    branches: [ "main" ]
```

Comment:
I would recommend to use
```
on:
  push:
    branches:
      - main
```

Reply:
Thanks @Ginxo ! I have updated the file.